src/ui/components/play-icon.ts

```diff
@@ -31,7 +33,9 @@ const icons = {
   'report-outline': reportOutline,
```

Comment:
external-outline + the SVG can probably be deleted.

:no_bell: Consider alphabetizing.

Reply:
"external-outline" is still used in the template dropdown.

Alphabetized the icon imports.